doc/source/examples.rst

```diff
@@ -276,6 +276,7 @@ generate an event file based on a raw corpus file and filter it:
     >>> from pyndl import preprocess
     >>> preprocess.create_event_file(corpus_file='doc/data/lcorpus.txt',
     ...                              event_file='doc/data/levent.tab.gz',
+    ...                              valid_symbols='a-zA-Z',
```

Comment:
I strongly opt against adding default characters to avoid any confusion. imho explicit is better than implicit in this case. 
a-zA-Z can (and is) suggested in the docs.